### DISCUSSION TITLE
Expand Support for More Than 8 Colors in osu!stable for New Combos like in osu!lazer

### DISCUSSION THREAD

**Discussion by Maks1mio** [ppy/osu#29167] [Ideas]
## Expand Support for More Than 8 Colors in osu!stable for New Combos like in osu!lazer

In the current version of osu!stable, the maximum number of colors available for new combos is limited to 8. This limitation restricts the ability to create diverse and visually appealing beatmaps. By expanding the support for more than 8 colors, similar to what is available in osu!lazer, mappers will have greater flexibility and creativity in designing their beatmaps.

### Demonstration
![image](https://github.com/user-attachments/assets/6380d6f0-384b-419a-a2f8-d1dd9c410d4f)
![image](https://github.com/user-attachments/assets/a0e345d1-ac4f-4120-9e4b-acbc37065ad0)

#### Why did I decide to suggest this?
I've never encountered this limitation, or maybe I didn't even notice it. But I was thinking of making different color combinations for each part based on the intended design in Figma under the storyboard.

https://github.com/user-attachments/assets/16018c9b-dd86-4063-bf45-9fc455ac1e6e

#### Important!
Yes, I am aware of this discussion (https://github.com/ppy/osu/discussions/14095), but I would like to have the same support on osu!stable, if not in the interface, where the maximum is 8 colors, then in the .osu file and then displayed in the osu!stable editor. So I'll write it here.

**Comment by Stoppedpuma** [comment]
Stable is feature locked and will not be receiving any new features or additions unless absolutely required, additionally this git repo is not for reporting things about stable.
